app/manage-projects/[projectSlug]/contributions/page.tsx

```diff
@@ -0,0 +1,11 @@
+"use client";
+
+import { Contributions } from "@/app/manage-projects/[projectSlug]/_features/contributions/contributions";
+
+export default function ManageProgramsContributionsPage({
+  params: { projectSlug },
+}: {
+  params: { projectSlug: string };
+}) {
+  return <Contributions projectSlug={projectSlug} />;
```

Comment:
why in a features ? We can do it directly inside the page to not create a deep architecture ?